### PR TITLE
Add CGFloat to TSDKCollectionObject Parsing

### PR DESCRIPTION
**Pull request by marquezzy**
Also added convenience property mapping description to teamFeeDescription

### CODE REVIEW

File: TeamSnapSDK/SDK/DataTypes/TSDKTeamFee.m [hunk 1]
```diff
@@ -16,4 +16,12 @@ + (NSString *)SDKType {
     return @"team_fee";
 }
 
+- (NSString *)teamFeeDescription {
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKTeamFee.m]: These shouldn't be needed, but there was a bug in the SDK that I fixed as part of the Team Media branch that caused the "description" postfix not to be handled properly on multi word types. The SDK tries to auto convert API objects sent as "description" to camelCaseClassDescription. 

https://github.com/teamsnap/teamsnap-SDK-iOS/commit/c255bf220ce6955f25188ddde914b48619b35d9a

You can either cherry pick that PR, or we'll need to remember to remove these two methods after my stuff is merged.

Comment (marquezzy) [inline TeamSnapSDK/SDK/DataTypes/TSDKTeamFee.m]: I tried to cherry-pick but I am getting a bad-object error. Can we hold off and remember to fix these methods? There is nothing broken with them at the moment.

**Comment by jrahaim** [comment]
Needs test for getCGFloat and setCGFloat

**Comment by jrahaim** [comment]
Looks good.. I don't see a test to make sure we have a value of 0.00 when the value from the server is null. Add that and we are good to go.. 

Welcome the the SDK!

**Pull request merged**